src/oaklib/implementations/semsimian/semsimian_implementation.py

```diff
@@ -113,3 +113,36 @@ def pairwise_similarity(
         sim.phenodigm_score = math.sqrt(sim.jaccard_similarity * sim.ancestor_information_content)
 
         return sim
+
+    def all_by_all_pairwise_similarity(
+        self,
+        subjects: Iterable[CURIE],
+        objects: Iterable[CURIE],
+        predicates: List[PRED_CURIE] = None,
+        min_jaccard_similarity: Optional[float] = None,
+        min_ancestor_information_content: Optional[float] = None,
+    ) -> Iterator[TermPairwiseSimilarity]:
+        """
+        Compute similarity for all combinations of terms in subsets vs all terms in objects
+
+        :param subjects:
+        :param objects:
+        :param predicates:
+        :return:
+        """
+        objects = list(objects)
+
+        for result in self.semsimian.all_by_all_pairwise_similarity(subjects,
+                                                                    objects,
```

Comment:
add the passthru for min thresholds - otherwise creating N^2 objects on the python side is expensive!